qpylib/log_qpylib.py

```diff
@@ -9,6 +9,7 @@
 
 APP_FILE_LOG_FORMAT = '%(asctime)s [%(module)s.%(funcName)s] [%(threadName)s] [%(levelname)s] - %(message)s'
 APP_CONSOLE_LOG_FORMAT = '%(asctime)s %(module)s.%(funcName)s: %(message)s'
+SYSLOG_APP_LOG_FORMAT = '1 %(asctime)s [app_id] %(message)s'
```

Comment:
Is the priority at the start optional?
App ID is a little ephemeral, a hostname like [app_name].[app_id] would be easier to understand and also ID the instance of the app if there's multiple instances. get_app_name can be called to get it, though you will need to convert any spaces to _

Reply:
This format doesn't comply with https://tools.ietf.org/html/rfc5424#section-6. The example given in the issue description is incomplete. There are a bunch of header fields missing. We need to decide what values are appropriate for each field in the header. I'll have a look and come up with a proposal.